### PR TITLE
coreclr: Automatic port of triple slash from System.Runtime.CompilerServices.R*-V*

### PR DESCRIPTION
**Pull request by carlossanlop**
Found some more triple slash comments that could be automatically ported from dotnet/coreclr
Adding area owner @Anipik to double check that the descriptions are correct. After this PR is checked-in, please proceed to manually document all the APIs that have missing documentation.
Adding @mairaw @rpetrusha for language review.

### CODE REVIEW

File: xml/System.Runtime.CompilerServices/TaskAwaiter`1.xml [hunk 1]
```diff
@@ -268,11 +268,16 @@
...
         <summary>Schedules the continuation action for the asynchronous task associated with this awaiter.</summary>
-        <remarks>To be added.</remarks>
+        <remarks>
+          <format type="text/markdown"><![CDATA[
+This method is intended for compiler user rather than use directly in code.
+          ]]></format>
+        </remarks>
         <exception cref="T:System.ArgumentNullException">
           <paramref name="continuation" /> is <see langword="null" />.</exception>
         <exception cref="T:System.InvalidOperationException">The awaiter was not properly initialized.</exception>
```
Comment (Anipik) [inline xml/System.Runtime.CompilerServices/TaskAwaiter`1.xml]: The unsafeOnCompleted doesnot throws InvalidOperationException, we need to remove this line.

Comment (carlossanlop) [reply xml/System.Runtime.CompilerServices/TaskAwaiter`1.xml]: @Anipik What about the methods called internally by UnsafeOnCompleted? Can you please confirm if they throw this exception?

Comment (carlossanlop) [reply xml/System.Runtime.CompilerServices/TaskAwaiter`1.xml]: @Anipik can you please double check if the exception is thrown by internally called methods?

Comment (Anipik) [reply xml/System.Runtime.CompilerServices/TaskAwaiter`1.xml]: yeah sure i will take a look at the internal methods as well

Comment (Anipik) [reply xml/System.Runtime.CompilerServices/TaskAwaiter`1.xml]: @carlossanlop I verified and the internal methods does not throw invalidOperationException

Comment (carlossanlop) [reply xml/System.Runtime.CompilerServices/TaskAwaiter`1.xml]:
```suggestion
```

**Comment by carlossanlop** [comment]
@rpetrusha Thanks. We addressed the last comment about InvalidOperationException (it needed to be removed). After the build finishes, we can get this merged.

**Pull request merged**